components/application-connectivity-validator/cmd/applicationconnectivityvalidator/options.go

```diff
@@ -18,8 +19,12 @@ type options struct {
 	eventMeshDestinationPath string
 	appRegistryPathPrefix    string
 	appRegistryHost          string
+	appName                  string
 	cacheExpirationMinutes   int
 	cacheCleanupMinutes      int
+	kubeConfig               string
+	masterURL                string
+	syncPeriod               time.Duration
 }
 
 func parseArgs() *options {
```

Comment:
Please add a description of new parameters to the file components/application-connectivity-validator/README.md